talon/quotations.py

```diff
@@ -464,13 +464,19 @@ def _extract_from_html(msg_body):
     if html_tree is None:
         return msg_body
 
-    cut_quotations = (html_quotations.cut_gmail_quote(html_tree) or
-                      html_quotations.cut_zimbra_quote(html_tree) or
-                      html_quotations.cut_blockquote(html_tree) or
-                      html_quotations.cut_microsoft_quote(html_tree) or
-                      html_quotations.cut_by_id(html_tree) or
-                      html_quotations.cut_from_block(html_tree)
-                      )
+    cut_quotations = False
+    try:
+        html_tree_pre_cut = deepcopy(html_tree)
```

Comment:
Why do we need to deepcopy twice?

Reply:
Not sure how to do it otherwise.

Reply:
why can't you just catch the exception, ignore it and proceed with whatever html_tree contains.

Reply:
I do not like the idea of doing 2 deep copies for 100% of cases just to fix a tiny minority of crappy emails.